README.adoc

```diff
@@ -22,40 +24,226 @@ Current
+        }
+
+        /*
+         * One may use server.run(); or create the server into a new thread
+         */
+        Thread serverThread = new Thread(server);
+
+        /*
+         * Run the server
+         */
+        serverThread.start();
+
+        /*
+         * Wait for startup, server is available for connections once it finished setup
+         */
+        try {
+            server.startup.waitForCompletion();
+            System.out.println("server started at port <" + listenPort + ">");
+        }
+        catch (InterruptedException interruptedException) {
+            throw new RuntimeException(interruptedException);
+        }
+
+        /*
+         * Send Hello, World! via rlp_01
+         */
+        new ExampleRelpClient(listenPort).send("Hello, World!");
+
+        /*
+         * Stop server
+         */
+        server.stop();
+
+        /*
+         * Wait for stop to complete
+         */
+        try {
+            serverThread.join();
+        }
+        catch (InterruptedException interruptedException) {
+            throw new RuntimeException(interruptedException);
+        }
+        System.out.println("server stopped at port <" + listenPort + ">");
+
+        /*
+         * Close the frameDelegate
+         */
+        try {
+            frameDelegate.close();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+
+    /**
+     * ExampleRelpClient using rlp_01 for demonstration
+     */
+    private class ExampleRelpClient {
+        private final int port;
+        ExampleRelpClient(int port) {
+            this.port = port;
+        }
+
+        public void send(String record) {
+            RelpConnection relpConnection = new RelpConnection();
+            try {
+                relpConnection.connect("localhost", port);
+            }
+            catch (IOException | TimeoutException exception) {
+                throw new RuntimeException(exception);
+            }
+
+            RelpBatch relpBatch = new RelpBatch();
+            relpBatch.insert(record.getBytes(StandardCharsets.UTF_8));
+
+            while (!relpBatch.verifyTransactionAll()) {
+                relpBatch.retryAllFailed();
+                try {
+                    relpConnection.commit(relpBatch);
+                }
+                catch (IOException | TimeoutException exception) {
+                    throw new RuntimeException(exception);
+                }
+            }
+            try {
+                relpConnection.disconnect();
+            }
+            catch (IOException | TimeoutException exception) {
+                throw new RuntimeException(exception);
+            }
+            finally {
+                relpConnection.tearDown();
+            }
         }
     }
 }
 ----
 
+In case one requires a separate handler for each connection that does not need to be thread-safe create new FrameDelegate in the Supplier<FrameDelegate>:
```

Comment:
> In case one requires a separate handler for each connection that does not need to be thread-safe create new FrameDelegate in the Supplier<FrameDelegate>:

-> "In case you need a separate handler for each connection but it doesn't need to be thread-safe, create a new `FrameDelegate` in the `Supplier<FrameDelegate>`:"

Reply:
Or even better: "If a separate handler is required for each connection which doesn’t need to be a thread-safe, create a new `FrameDelegate` in the `Supplier<FrameDelegate>`:"